src/np_codeocean/upload.py

```diff
@@ -336,7 +345,8 @@ def create_codeocean_upload(session: str | int | np_session.Session,
         session = np_session.Session(session)
         root = np_session.NPEXP_PATH / 'codeocean' / session.folder
         behavior = np_config.normalize_path(root / 'behavior')
-        behavior_videos = behavior.with_name('behavior-videos')
+        if modality in ('ecephys', ):
```

Comment:
why only for ecephys? surely it should be for both

Reply:
Addressed here : 2db595654b38665d5297fd4d648d8af9082ccb38